docs/source/content/deep_learning/multilayer.rst

```diff
@@ -0,0 +1,151 @@
+##########
+Multi-layer Perceptron
+##########
+
+.. contents::
+  :local:
+  :depth: 3
+
+********
+Overview
+********
+
+A multilayer perceptron (MLP) is a deep, artificial **neural network**.
+A *neural network* is comprised of layers of nodes which activate at various
+levels depending on the previous layer's nodes.
+When thinking about neural networks, it may be helpful to isolate your thinking to a single node in the network.
+
+Multilayer perceptron refers to a neural network with at least three layers of nodes, an input layer, some
+number of intermediate layers, and an output layer.
+Each node in a given layer is connected to every node in the adjacent layers.
+The input layer is just that, it is the way the network takes in data.
+The intermediate layer(s) are the computational machine of the network, they
+actually transform the input to the output.
+The output layer is the way that results are obtained from the neural network.
+In a simple network where the responses are binary, there would likely be only one node in the output layer,
+which outputs a probability like in 'logistic regression'_.
```

Comment:
This link isn't showing up

Reply:
Fixed, needed a special single quote character (i think)